.env.example

```diff
@@ -1,5 +1,6 @@
 RAILS_ENV=development
 RAILS_LOG_TO_STDOUT=true
+LOCAL_TEST=
```

Comment:
nit: I don't find this variable to be very self-explanatory. Since it only controls one thing right now, I would maybe just call it `DISABLE_SSL`, and then we could make it more general at some point if we want the idea of local testing to encompass more things.